packages/api-client/tests/secret.spec.ts

```diff
@@ -12,6 +12,7 @@ describe('Secret Controller Tests', () => {
   let workspaceSlug: string | null
   let environmentSlug: string | null
   let secretSlug: string | null
+  let environment
```

Comment:
add type

Reply:
or is just type:any fine ?? cause in other files its so

Reply:
Ig environment: Environment ,imported from '@api-client/types/environment.types' works

Reply:
you are right, that almost left my mind. do one thing, don't compare the ID, just compare the slugs. and also, just store the `environmentSlug` in here

Reply:
You dont have to store the environment id. we are already storing the environment slug so you can just do that comparison and drop the id comparison.